<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTallenna">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tallenna</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Anna nimesi:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEtunimi">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblViesti">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hei.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAvaaLomake2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Avaa lomake 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSukunimi">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sukunimi</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
